tools/clang/test/HLSLFileCheck/hlsl/types/modifiers/groupshared/groupshared_shadermodels.hlsl

```diff
@@ -28,24 +29,30 @@ int4 getit()
   return foo;
 }
 
+// LIBCHK: error: Thread Group Shared Memory not supported from non-compute entry points.
+// LIBCHK: error: Thread Group Shared Memory not supported from non-compute entry points.
+// LIBCHK: error: Thread Group Shared Memory not supported from non-compute entry points.
 // LIBCHK: error: Thread Group Shared Memory not supported from non-compute entry points.
 [shader("vertex")]
 float4 VSMain(uint ix : SV_VertexID) : OUT {
-  output[ix] = getit();
+  output[ix] = foo;
```

Comment:
This is just because it's more convenient to see the correct line number when identifying the groupshared memeory